controllers/instanceController.js

```diff
@@ -40,54 +84,110 @@ const createBaaS = (req, res, next) => {
       {
         ParameterKey: 'AppServerLG',
         ParameterValue: process.env.AppServerLG
+      },
+      {
+        ParameterKey: 'ListenerRulePriority',
+        ParameterValue: String(rulePriorityValue)
+      },
+      {
+        ParameterKey: 'TargetGroupName',
+        ParameterValue: TargetGroupName
+      },
+      {
+        ParameterKey: 'ClusterName',
+        ParameterValue: ClusterName
+      },
+      {
+        ParameterKey: 'NewNameSpace',
+        ParameterValue: stackName
+      },
+      {
+        ParameterKey: 'RoutingPath',
+        ParameterValue: RoutingPath
+      },
+      {
+        ParameterKey: 'AppId',
+        ParameterValue: appId
       }
     ],
     Capabilities: ['CAPABILITY_NAMED_IAM']
   };
+};
 
+const createBaaS = async (req, res, next) => {
+  const appId = nanoid.nanoid();
```

Comment:
As mentioned in the main comment. Feel free to revert this if you don't like the approach.